### DISCUSSION TITLE
Housing Tag data for the Housing Browser API

### DISCUSSION THREAD

**Discussion by BusterBrown1218** [HypixelDev/PublicAPI#655] [Ideas]
Recently an endpoint was added to the API for the Housing Browser, which is really helpful.
The problem is that it appears to lack tag data for housings (Roleplay, General, Parkour, Maze, Chat, Guild Hangout, Build, or Minigame)

It would be nice to add with a simple "tag" property on each Housing object.
Thanks!

**Comment by penpen-uwu** [comment]
what about the house icon

**Reply by ixnoahlive** [reply]
With animated icons wip, i don't see this coming soon

**Reply by penpen-uwu** [reply]
cant it just be text saying which icon is chosen? how would animated icons make it hard to implement

**Reply by ixnoahlive** [reply]
Sure that works, but if you want to properly display them on the client with a mod like what I assume Buster is doing based on what they posted about, NBT would be needed if skulls were added.

**Comment by ixnoahlive** [comment]
bump?
